terraform/modules/common/ami/outputs.tf

```diff
@@ -0,0 +1,3 @@
+output "ami_id" {
+  value = "${data.aws_ami.source.id}"
+}
```

Comment:
I don't think we need separate files for variables and outputs. @surminus explained why better than I could: https://surminus.github.io/post/terraform-structures-and-layouts/ (see section "variables and outputs")